stasis/README.md

```diff
@@ -0,0 +1,13 @@
+# Chainlink External Adapter for Stasis
+
+## Input Params
+
+_None_
+
+## Output
+
+```json
```

Comment:
The output here should probably be the output from the Chainlink EA, not the Stasis API response.

One way to get this would be to log `JSON.stringify(data)` in the `'successful calls @integration'` tests.

Reply:
Ok that makes sense